### ISSUE TITLE
mingw-w64-boost-1.68.0-2

### ISSUE THREAD

**Issue by xantares** [Martchus/PKGBUILDs#73]
hi,
revision 2 fixes stdcxx14 and is necessary to link with the default gcc std=c++14 mode
could you update it in your repo ?
xan.

**Comment by Martchus** [comment]
added the package

**Comment by xantares** [comment]
thanks

**Comment by xantares** [comment]
also, do the packages need a global rebuild after the crt 6.0.0 update ?

**Comment by Martchus** [comment]
I don't think so. The API/ABI of the mingw-w64 headers and CRT is supposed to match the stable API/ABI provided by the Windows platform itself and as such should be stable as well.

I also tested some of my apps recently and had only Qt and things higher in the dependency stack rebuilt. I didn't notice any issues (at least when testing via wine).

Some build error due to subtle changes in mingw-w64 6.0.0 are to be expected (eg. when updating Qt to 5.11.2 I had to add yet another 2 patches). But those things can be fixed as soon the package needs to be rebuilt anyways.
